components/admin-dashboard/whitelisted-addresses.ts

```diff
@@ -1,4 +1,3 @@
-export const whitelistedAddresses = process.env
-  .NEXT_PUBLIC_WHITELISTED_ADDRESSES
-  ? process.env.NEXT_PUBLIC_WHITELISTED_ADDRESSES.split(",")
-  : [];
+import { env } from "@/config/env.mjs";
+
+export const whitelistedAddresses = env.NEXT_PUBLIC_WHITELISTED_ADDRESSES;
```

Comment:
@EsterB12 do you not need the .split() anymore? is this in the env.mjs file?

Reply:
yes, its in the env.mjs!